server/channels/store/storetest/draft_store.go

```diff
@@ -218,6 +219,82 @@ func testDeleteDraft(t *testing.T, rctx request.CTX, ss store.Store) {
 	})
 }
 
+func testPermanentDeleteDraftsByUser(t *testing.T, rctx request.CTX, ss store.Store) {
+	t.Run("should delete all drafts for a given user", func(t *testing.T) {
+		userId := model.NewId()
+		channel1Id := model.NewId()
+		channel2Id := model.NewId()
+
+		member1 := &model.ChannelMember{
+			ChannelId:   channel1Id,
+			UserId:      userId,
+			NotifyProps: model.GetDefaultChannelNotifyProps(),
+		}
+
+		member2 := &model.ChannelMember{
+			ChannelId:   channel2Id,
+			UserId:      userId,
+			NotifyProps: model.GetDefaultChannelNotifyProps(),
+		}
+
+		_, err := ss.Channel().SaveMember(rctx, member1)
+		require.NoError(t, err)
+
+		_, err = ss.Channel().SaveMember(rctx, member2)
+		require.NoError(t, err)
+
+		draft1 := &model.Draft{
+			CreateAt:  model.GetMillis(),
+			UpdateAt:  model.GetMillis(),
+			UserId:    userId,
+			ChannelId: channel1Id,
+			Message:   "draft1",
+		}
+
+		draft2 := &model.Draft{
+			CreateAt:  model.GetMillis(),
+			UpdateAt:  model.GetMillis(),
+			UserId:    userId,
+			ChannelId: channel2Id,
+			Message:   "draft2",
+		}
+
+		_, err = ss.Draft().Upsert(draft1)
+		require.NoError(t, err)
+
+		_, err = ss.Draft().Upsert(draft2)
+		require.NoError(t, err)
+
+		// Delete draft for the user
```

Comment:
Can you call `GetDraftsForUser` and verify the non-zero count before deleting them? That way we can see that the count from from something to 0.